pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Execute a command on a group of hosts:
```

Comment:
**New changes**
- Use complete sentences that start with verbs
- Use phrasing similar to the [official docs](http://docs.ansible.com/ansible/modules_by_category.html)

RE: command simplicity, please see [my comments](https://github.com/tldr-pages/tldr/pull/1153#discussion_r88134311)